src/main/resources/config.json

```
@@ -1,12 +1,19 @@
 {
   "token": "",
   "database": {
-    "host": "localhost",
+    "host": "database",
```

```suggestion
    "host": "localhost",
```
Seems to be more logical to pass localhost or 0.0.0.0 so that you are able to test stuff without configuration.

---

Yeah did this to sync docker and template config. Didn't wanted to maintain both, but maybe we have to.

---

`ln -s` ?